test/config/type/function-with-env/function-with-env.test.js

```diff
@@ -18,6 +25,34 @@ describe('function configuration', () => {
         // Should generate the appropriate files
         expect(existsSync(resolve(__dirname, './bin/dev.js'))).toBeTruthy();
     });
+    it('Supports passing string in env', () => {
+        const { stderr, stdout } = run(__dirname, [
+            '--env',
+            'environment=production',
+            '--env',
+            'app.title=Luffy',
+            '-c',
+            'webpack.env.config.js',
+        ]);
+        expect(stderr).toBeFalsy();
+        expect(stdout).toBeTruthy();
+        // Should generate the appropriate files
+        expect(existsSync(resolve(__dirname, './bin/Luffy.js'))).toBeTruthy();
+    });
+    it('Supports long nested values in env', () => {
+        const { stderr, stdout } = run(__dirname, [
+            '--env',
+            'file.name.is.this=Atsumu',
+            '--env',
+            'environment=production',
+            '-c',
+            'webpack.env.config.js',
+        ]);
+        expect(stderr).toBeFalsy();
+        expect(stdout).toBeTruthy();
+        // Should generate the appropriate files
+        expect(existsSync(resolve(__dirname, './bin/Atsumu.js'))).toBeTruthy();
+    });
```

Comment:
Can you add test:
- `--env name=foo=bar`
- `--env` (no flags)
- `--env name.` (dot at the end)
- `--env name.=test` (dot at the end)

Reply:
Done